app/views/questions/show.html.erb

```diff
@@ -55,6 +55,7 @@
   <%= form_for [@question, @answer], url: { controller: "answers", action: "create" } do |f| %>
     <%= f.hidden_field :question, :value => params[:id] %>
     <%= f.text_area :body, placeholder: "Enter text" %>
+    <%= show_errors(@answer, :body) %>
```

Comment:
Man, you're not kidding that it's a PITA to implement. I wouldn't have expected it to be necessary for every single item in a form, but it definitely makes sense. Thanks for taking care of this @TheKaterTot !